translations/en-us.yaml

```diff
@@ -3771,6 +3771,14 @@ clusterNew:
         label: CNI Type
       apiEndpoint:
         label: Kubernetes API Endpoint Subnet
+    numWorkerNodes:
+      label: Node Count
+      help: The number of nodes in the node pool
+    vcn:
+      quickResources:
+        header: 'New resources include:'
+        k8s: Kubernetes cluster
```

Comment:
unsure these belong, the description of items was meant to be what you'd get for quick create networking (new vcn and all other things).